subprojects/tree-sitter-xml-parser.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-git]
```

Comment:
wrap-git is forbidden in the wrapdb

Reply:
Should I use https://github.com/unhammer/tree-sitter-xml/archive/refs/heads/master.zip and decide a fake version? There's no official XML parser implementation

https://github.com/tree-sitter/tree-sitter/issues/266